docs/release/trg-7/trg-7-07.md

```diff
@@ -2,9 +2,15 @@
 title: TRG 7.07 - Legal notice for documentation
 ---
 
-| Status | Created     | Post-History  |
-|--------|-------------|---------------|
-| Draft  | 13-Apr-2023 | New           |
+:::caution
+This version is deprecated.
+Please see new drafts [TRG 7.07](/docs/release/trg-0/trg-7-07) and [TRG 7.08](/docs/release/trg-0/trg-7-08) for review.
+:::
+
+| Status      | Created     | Post-History    |
+|-------------|-------------|-----------------|
+| Draft       | 13-Apr-2023 | New             |
+| Deprecated  | 19-Jul-2023 | Updated version in drafts|          |
```

Comment:
I think we are adding the 'latest' change to the top of the table, instead of the bottom.
At least the [overall changelog](https://eclipse-tractusx.github.io/docs/release/trg-0/) does it like that.

Also not critical. At least I think, we can change that, when it is moved to the 'live' section after the feedback period